### PR TITLE
added tests to verify product info on view page and customer portal

### PR DESCRIPTION
**Pull request by pbadak**

### CODE REVIEW

File: suites/test_publish_module.py [hunk 1]
```diff
@@ -59,6 +68,21 @@ def publish_module(self):
         utilities.wait(5)
         utilities.click_element_by_css_selector(
             self.driver, locators.MODULE_DISPLAY_PUBLISH_BUTTON_CSS)
+        time.sleep(5)
+        # get updated date in variable and covert into desired format
+        updated_date_module_page_txt = (utilities.get_text_by_css(
```
Comment (anarang) [inline suites/test_publish_module.py:73]: Change the keyword updated_date to uploaded_date in sync with the UI.

File: suites/test_publish_module.py [hunk 2]
```diff
@@ -59,6 +68,21 @@ def publish_module(self):
...
+            updated_date_module_page_txt, "%a %b %d %Y %H:%M:%S %Z%z")
+        test_publish_module.updated_date_module_page = datetime.datetime.strftime(
+            test_publish_module.updated_date_module_page, "%d %B %Y")
+        # get published date in variable and convert into desired format
+        published_date_module_page_txt = (utilities.get_text_by_css(
+            self.driver, locators.PUBLISHED_DATE_CSS)).rstrip('\n Version 1')
+        test_publish_module.published_date_module_page = datetime.datetime.strptime(
+            published_date_module_page_txt, "%a %b %d %Y %H:%M:%S %Z%z")
+        test_publish_module.published_date_module_page = datetime.datetime.strftime(
+            test_publish_module.published_date_module_page, "%d %B %Y")
```
Comment (anarang) [inline suites/test_publish_module.py:85]: Add spacing before and after this block of code.
Add a line of code somewhere whic prints which module is being published.
Add an lcc.log_info("Captured the uplaoded time as %s" % var)
Same for published time.

File: suites/test_publish_module.py [hunk 3]
```diff
@@ -59,6 +68,21 @@ def publish_module(self):
         utilities.wait(5)
         utilities.click_element_by_css_selector(
             self.driver, locators.MODULE_DISPLAY_PUBLISH_BUTTON_CSS)
+        time.sleep(5)
+        # get updated date in variable and covert into desired format
+        updated_date_module_page_txt = (utilities.get_text_by_css(
+            self.driver, locators.UPDATED_DATE_CSS))
+        test_publish_module.updated_date_module_page = datetime.datetime.strptime(
```
Comment (anarang) [inline suites/test_publish_module.py:75]: change the variable name as you are overriding one variable too many times.

Comment (anarang) [reply suites/test_publish_module.py:75]: same for the publish variable too.

**Comment by pbadak** [comment]
code modified as per above review comments

**Pull request merged**